SEED-platform/seed
# Add Cycles Views to Swagger, APIv2

#### Any background context you want to provide?
The cycles API wasn't yet in Swagger/APIv2 land.  

#### What's this PR do?

* Move cycles to API v2
* Expose cycles in Swagger
* Ignores the cycles endpoints in the trailing slash unit test because it seems it doesn't matter...

#### How should this be manually tested?

* Make sure endpoints show up on Swagger
* Make sure endpoints work on Swagger
* Unit tests should pass
* Monkey test front end to find unexpected issues

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/133876307
